jep/7/README.adoc

```diff
@@ -100,6 +149,8 @@ In a Jenkins developers mailing list thread proposing the deprecation of `ruby-r
 
 Therefore this JEP proposed the deprecation of ruby-runtime, and to suspend its distribution via Jenkins project update sites.
 
+The same reasoning applies to the Jenkins Python SDK.
```

Comment:
Please reference a single change, hack, or workaround we had to do to make `python-wrapper` work.

Reply:
```suggestion
The same concerns apply to the Jenkins Python SDK.
Although we have not experienced issues with that due to the low adoption, potentially it may lead to the core maintenance overhead.
Low adoption and lack of interest in Jenkins plugin development for Python suggests that we can just remove the support as a part of this JEP.
```

Would it work?